crates/rolldown/src/ecmascript/format/iife.rs

```diff
@@ -65,14 +78,79 @@ pub fn render_iife(
   }
 
   // iife wrapper end
-  concat_source.add_source(Box::new(RawSource::new(format!(
-    "}})({});",
-    if has_exports && named_exports { "{}" } else { "" }
-  ))));
+  if invoke {
+    concat_source.add_source(Box::new(RawSource::new(format!("}})({output_args});"))));
```

Comment:
Why here add `invoke` condiction, look like it's unnecessary, the iif wrapper always need to invoke.

Reply:
In `iife`, it is unnessary, but in `amd`, `umd`, it won't be invoked with parameters. I think this function can be reused in generating `amd`, or `umd` function.
I think merging the function to `wrap.rs` is better, but it's better to refactor it in a separate PR, imo.

Reply:
Ok. I see. Maybe you could add some comment at now.

Reply:
Comment added now.